catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/operation/UnitTestPostgreSqlTableOperations.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.postgresql.operation;
+
+import com.datastrato.gravitino.rel.indexes.Index;
+import com.datastrato.gravitino.rel.indexes.Indexes;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/** Unit test for {@link PostgreSqlTableOperations}. */
+public class UnitTestPostgreSqlTableOperations {
```

Comment:
I suggest to rename this to `TestPostgreSqlTableOperations` to keep consistent with other UTs